addOns/browserView/src/main/java/org/zaproxy/zap/extension/browserView/ExtensionHttpPanelBrowserView.java

```diff
@@ -95,8 +95,11 @@ private static boolean isJavaFxAvailable() {
             javafx.embed.swing.JFXPanel unused = new javafx.embed.swing.JFXPanel();
             return true;
         } catch (Throwable e) {
-            LOGGER.warn("Unable to use JavaFX:", e);
+            LOGGER.warn("Unable to use JavaFX: " + e.getMessage());
             System.setProperty(ZAP_JAVAFX_INIT_FAILED_SYSTEM_PROPERTY, "true");
+            if (LOGGER.isDebugEnabled()) {
+                LOGGER.warn(e);
```

Comment:
- Why not debug? Looks odd check debug and then warn.
 - Could be done immediately after the warn.

Reply:
Ok